tools/serverpod_cli/test/database/extentions/pgsql_code_generation_test.dart

```diff
@@ -2,9 +2,12 @@ import 'package:recase/recase.dart';
 import 'package:serverpod_cli/analyzer.dart';
 import 'package:serverpod_cli/src/database/create_definition.dart';
 import 'package:serverpod_cli/src/test_util/builders/class_definition_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/generator_config_builder.dart';
 import 'package:test/test.dart';
 
 void main() {
+  var config = GeneratorConfigBuilder().build();
```

Comment:
Suggestion: Remove the generator config builder and define what we need directly.